### ISSUE TITLE
NoClassDefFoundError while running marytts-server

### ISSUE THREAD

**Issue by zenny** [marytts/marytts#512]
I am getting the following that seems recurrence of https://github.com/marytts/marytts/issues/365 :

```
/local/mary/marytts/target/marytts-5.1.2/bin$ sudo -u mary ./marytts-server
java version "1.7.0_95"
OpenJDK Runtime Environment (IcedTea 2.6.4) (7u95-2.6.4-1~deb8u1)
OpenJDK 64-Bit Server VM (build 24.95-b01, mixed mode)

MARY server 5.1.2 starting as a HTTP server...Exception in thread "main" java.lang.NoClassDefFoundError: com/ibm/icu/text/RuleBasedNumberFormat
        at marytts.language.fr.Preprocess.<init>(Preprocess.java:37)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
        at java.lang.Class.newInstance(Class.java:383)
        at marytts.util.MaryRuntimeUtils.instantiateObject(MaryRuntimeUtils.java:130)
        at marytts.modules.ModuleRegistry.instantiateModule(ModuleRegistry.java:87)
        at marytts.server.Mary.startModules(Mary.java:130)
        at marytts.server.Mary.startup(Mary.java:296)
        at marytts.server.Mary.startup(Mary.java:203)
        at marytts.server.Mary.main(Mary.java:513)
Caused by: java.lang.ClassNotFoundException: com.ibm.icu.text.RuleBasedNumberFormat
        at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
        ... 12 more
Exception in thread "Thread-1" java.lang.IllegalStateException: MARY system is not running
        at marytts.server.Mary.shutdown(Mary.java:385)
        at marytts.server.Mary$3.run(Mary.java:288)

```

**Comment by psibre** [comment]
This is solved, please see #274.

**Comment by zenny** [comment]
@psibre I just downloaded the tarred 5.1.2 and ran ./marytts and it worked well!

In the meantime, I just finished listening to your very informative interview at http://hackerpublicradio.org/eps/hpr1599.spx Good interview, Ingmar! :+1:
